### PR TITLE
[IMP] sale_commission: updated some views

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: sale_commission/models/sale_order.py [hunk 1]
```diff
@@ -84,7 +83,7 @@ def onchange_agent(self):
     @api.depends('sale_line.price_subtotal')
     def _compute_amount(self):
         for line in self:
-            line.amount = 0.0
+            amount = 0.0
```
Comment (pedrobaeza) [inline sale_commission/models/sale_order.py]: This is not needed, as these values are cached and you don't have performance loss in this case.

File: sale_commission/views/settlement_view.xml [hunk 2]
```diff
@@ -55,35 +56,41 @@
...
+                    <sheet string="Settlement">
+                        <div class="oe_title" >
+                            <h1><field name="agent" /></h1>
+                        </div>
                         <group>
-                            <field name="date_from"/>
-                            <field name="invoice" attrs="{'invisible': [('state', '!=', 'invoiced')]}"/>
+                            <group>
+                                <field name="date_from"/>
+                                <field name="invoice"
```
Comment (pedrobaeza) [inline sale_commission/views/settlement_view.xml]: Please don't make cosmetic changes that makes the diff very large

File: sale_commission/__init__.py [hunk 3]
```diff
@@ -4,3 +4,4 @@
 
 from . import models
 from . import wizard
+from .hooks import set_commission_total
```
Comment (pedrobaeza) [inline sale_commission/__init__.py]: Newline at the end

File: sale_commission/hooks.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    """
+    When there are to many sale.order created it takes a very long time to
+    create the computed field commission_total, in order to ease the
+    installation we create the field manually.
+    :param cr: database cursor
+    :return: void
+    """
+    cr.execute("""
+        ALTER TABLE sale_order
+        ADD COLUMN commission_total double precision;""")
```
Comment (pedrobaeza) [inline sale_commission/hooks.py:18]: You must make the computation also by SQL

Comment (oihane) [inline sale_commission/hooks.py:18]: it depends on fields that will be 0, so commission_total will be 0.

Comment (pedrobaeza) [inline sale_commission/hooks.py:18]: OK, please put a comment about that then

File: sale_commission/views/settlement_view.xml [hunk 5]
```diff
@@ -55,35 +55,40 @@
                                 name="action_cancel" />
                         <field name="state" widget="statusbar"/>
                     </header>
-                    <h1><field name="agent" /></h1>
-                    <group>
+                    <sheet string="Settlement">
```
Comment (pedrobaeza) [inline sale_commission/views/settlement_view.xml:58]: Why these changes? This was recently changed in https://github.com/OCA/commission/pull/92, so you shouldn't remove it

File: sale_commission/hooks.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+# © 2016 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+
+def set_commission_total(cr):
+    """
+    When there are to many sale.order created it takes a very long time to
+    create the computed field commission_total, in order to ease the
+    installation we create the field manually.
+    As the value of it will be calculated from data that is not created it's
```
Comment (pedrobaeza) [inline sale_commission/hooks.py]: As its value will be computed from data that still doesn't exist, its value will be always 0, so there's no need to be filled.

File: sale_commission/views/settlement_view.xml (hunk 5 above)
Comment (oihane) [inline sale_commission/views/settlement_view.xml:58]: I do not remove it, I just changed the formatting adding a sheet.

Comment (pedrobaeza) [inline sale_commission/views/settlement_view.xml:58]: OK

**Comment by oihane** [comment]
@pedrobaeza all changes done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by ivantodorovich** [comment]
:+1:  looks good
